src/pages/HomePage.tsx

```diff
@@ -1,14 +1,273 @@
-import { type JSX, useEffect } from "react";
-import { useNavigate } from "react-router";
+import { VirtuosoMasonry } from "@virtuoso.dev/masonry";
```

Comment:
<img width="279" height="126" alt="Image" src="https://github.com/user-attachments/assets/63ab4bdc-e3a0-4c8f-bcd5-561b4460f8c7" />

Maybe hide this for single instance setups?